src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -24,14 +28,18 @@ public OverseasEntitySubmissionDtoValidator(EntityDtoValidator entityDtoValidato
         this.overseasEntityDueDiligenceValidator = overseasEntityDueDiligenceValidator;
     }
 
-    public Errors validate(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errs, String loggingContext) {
-        entityDtoValidator.validate(overseasEntitySubmissionDto.getEntity(), errs, loggingContext);
-        presenterDtoValidator.validate(overseasEntitySubmissionDto.getPresenter(), errs, loggingContext);
+    public Errors validate(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
+        entityDtoValidator.validate(overseasEntitySubmissionDto.getEntity(), errors, loggingContext);
+        presenterDtoValidator.validate(overseasEntitySubmissionDto.getPresenter(), errors, loggingContext);
         if(Objects.nonNull(overseasEntitySubmissionDto.getOverseasEntityDueDiligence())) {
-            overseasEntityDueDiligenceValidator.validate(overseasEntitySubmissionDto.getOverseasEntityDueDiligence(), errs, loggingContext);
+            overseasEntityDueDiligenceValidator.validate(overseasEntitySubmissionDto.getOverseasEntityDueDiligence(), errors, loggingContext);
         }
-        return errs;
+        validateBeneficialOwnersStatement(overseasEntitySubmissionDto.getBeneficialOwnersStatement(), errors, loggingContext);
+        return errors;
     }
 
-
+    private boolean validateBeneficialOwnersStatement(BeneficialOwnersStatementType beneficialOwnersStatement, Errors errors, String loggingContext) {
```

Comment:
i'd move the BO statement validation into it's own component as there is a requirement to be able to validate each page separately if we need to (and BO statement has it's own page)

Reply:
I was looking at this form the point of view of the api model structure, are you sure?

Reply:
yes, it will give us more flexibility if we need to validate by page (when we do save and resume), the api structure should also be loosely based on page.

Reply:
So yes, I agree on this point. I'm wondering @mattch1 why you've even written the BO statement validation code in this PR if, as @markpit says, it has its own page and this PR is for the 'Have any registrable beneficial owners been identified?' screen?

Reply:
Moved code into its own class.